pkg/project/auth/reviewer.go

```diff
@@ -73,7 +74,9 @@ func (r *authorizerReviewer) Review(namespaceName string) (Review, error) {
 		ResourceRequest: true,
 	}
 
-	users, groups, err := r.policyChecker.GetAllowedSubjects(attributes)
+	subjects, err := r.policyChecker.AllowedSubjects(attributes)
```

Comment:
Add comments for places where its OK to "ignore" errors.

Reply:
Please, see my comments at https://github.com/openshift/origin/pull/16034/files#diff-2004baa088cb0f8c9950e3c41b695797R67. I'm confused about what you meant.

Reply:
Something like this:

```go
func (r *authorizerReviewer) Review(namespaceName string) (Review, error) {
	attributes := kauthorizer.AttributesRecord{
		Verb:            "get",
		Namespace:       namespaceName,
		Resource:        "namespaces",
		Name:            namespaceName,
		ResourceRequest: true,
	}

	// err is non fatal and partial success is possible
	subjects, err := r.policyChecker.AllowedSubjects(attributes)
	// expandSubjectError is non fatal and partial success is possible
	users, groups, expandSubjectError := authorizationutil.ExpandSubjects(namespaceName, subjects)

	review := &defaultReview{
		users:  users.List(),
		groups: groups.List(),
	}

	if err != nil {
		review.evaluationError = err.Error()
	}
	if expandSubjectError != nil {
		review.evaluationError += " " + expandSubjectError.Error()
	}

	return review, nil
}
```